### DISCUSSION TITLE
Create node which can browse files

### DISCUSSION THREAD

**Discussion by jonahpearl** [bonsai-rx/bonsai#1268] [Q&A]
Hi there,

Sorry if this has been asked before but I can't find anything about it. I'm working on a workflow that can auto-set filenames of multiple files, so that I only have to change the file path in one place per experiment. (I also have different extensions for different videos which have to be set correctly, etc...so I've encapsulated all of that into a group workflow.) I'd like to be able to set the desired save directory by opening up a system dialog box, a la what happens when you click the 3 dots on a `FileName` property on a sink. Currently, I just copy and paste the path into a `String` node. Any thoughts on whether it's possible to get a generic file browser? Thanks! Example file attached.
[basler_subjAutoSet_test_nested.zip](https://github.com/bonsai-rx/bonsai/files/10863351/basler_subjAutoSet_test_nested.zip)

**Comment by bruno-f-cruz** [comment]
Check #1204 for a potential solution

**Reply by jonahpearl** [reply]
Looks like what I want 👍 . Any idea on timeline betw merging + getting into a release? Not going to mess with trying to compile a patched version myself!

**Reply by bruno-f-cruz** [reply]
you should be able to use it directly as an extension:
https://bonsai-rx.org/docs/tutorials/scripting.html
